libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyManagerTests.java

```diff
@@ -236,8 +268,8 @@ private static Class<?> makeClassInItsOwnModule() throws IOException, ClassNotFo
         return layer.findLoader("org.example.plugin").loadClass("q.B");
     }
 
-    private static PolicyManager policyManagerWithEntitlementsModule(Module entitlementsModule) {
-        return new PolicyManager(createEmptyTestServerPolicy(), List.of(), Map.of(), c -> "test", entitlementsModule);
+    private static PolicyManager policyManagerWithEntitlementsModule(Module agentsModule, Module entitlementsModule) {
```

Comment:
nit, the method name is no longer accurate. Could this just be `policyManager(...)` now?

Reply:
Changed.